support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -222,8 +253,15 @@ export function SupporterPlusLandingPage({
 						<Box
 							cssOverrides={optimisedMobileLayout ? shorterBoxMargin : css``}
 						>
-							<AmountAndBenefits />
+							<AmountAndBenefits hideOneOff={supportOnceDisplay} />
 						</Box>
+						<CheckoutSupportOnceContainer
```

Comment:
Instead of passing `supportOnceDisplay` all the way down to the component tree to make the decision whether to render it or not we could do so here, eg.

```
(supportOnceDisplay && <CheckoutSupportOnceContainer>)
```

This simplifies the prop signatures for `CheckoutSupportOnceContainer` and `CheckoutSupportOnce`, and it means we're executing less code needlessly as we're making the further up the component tree without having to needlessly run code for `CheckoutSupportOnceContainer` and `CheckoutSupportOnce` if `supportOnceDisplay` is `false`.

Reply:
Yes should spotted that, have seen this done many times reducing need for unnecessary props. Have corrected/simplified this as suggested.